### PR TITLE
First Time User

### PR DESCRIPTION
**Pull request by ThomasHEaton**

### CODE REVIEW

File: app/src/main/java/ca/carleton/gcrc/n2android_mobile1/activities/AddConnectionActivity.java [hunk 1]
```diff
@@ -124,11 +161,11 @@ public void createConnection(){
             }
         }
 
-        Log.v(TAG, "Connection " + connectionName + "/" + url + "/" + userName + "/" + userPassword);
+        Log.v(TAG, "Connection " + "/" + url + "/" + userName + "/" + userPassword);
 
         try {
             ConnectionInfo info = new ConnectionInfo();
-            info.setName(connectionName);
+            info.setName("Atlas Name");
```
Comment (grrrian) [inline app/src/main/java/ca/carleton/gcrc/n2android_mobile1/activities/AddConnectionActivity.java:168]: are we displaying this name? Is the plan to replace it in the future?

File: app/src/main/java/ca/carleton/gcrc/n2android_mobile1/activities/AddConnectionActivity.java [hunk 2]
```diff
@@ -85,14 +122,14 @@ protected void onDestroy() {
...
-            }
-        }
+//        String connectionName = null;
+//        {
+////            View editTextView = findViewById(R.id.connectionName);
+////            if( editTextView instanceof EditText){
+////                EditText editText = (EditText)editTextView;
+////                connectionName = editText.getText().toString();
+////            }
+//        }
```
Comment (grrrian) [inline app/src/main/java/ca/carleton/gcrc/n2android_mobile1/activities/AddConnectionActivity.java:132]: is there value in keeping this around?

File: app/src/main/java/ca/carleton/gcrc/n2android_mobile1/activities/AddConnectionActivity.java (hunk 1 above)
Comment (ThomasHEaton) [reply app/src/main/java/ca/carleton/gcrc/n2android_mobile1/activities/AddConnectionActivity.java:168]: Yes, there is a card around getting the name from the Atlas. The next issue is related to that as well. The below comment section could be removed in the meantime

File: app/src/main/java/ca/carleton/gcrc/n2android_mobile1/activities/MainActivity.java [hunk 3]
```diff
@@ -151,6 +180,47 @@ public void drawList() {
...
+        boolean enabled = urlTextView.getText().length() != 0 && userNameTextView.getText().length() != 0 &&
+                passwordTextView.getText().length() != 0;
+
+        if (enabled) {
+            Button button = findViewById(R.id.button_create);
+            button.setEnabled(true);
+        } else {
+            Button button = findViewById(R.id.button_create);
+            button.setEnabled(false);
+        }
```
Comment (grrrian) [inline app/src/main/java/ca/carleton/gcrc/n2android_mobile1/activities/MainActivity.java:221]: this if/else block can be reduced to:
```
Button button = findViewById(R.id.button_create);
button.setEnabled(enabled);
```

**Review by grrrian — APPROVED** [review]

**Pull request merged**